### PR TITLE
Fix Examples JavaDoc

### PR DESCRIPTION
**Pull request by andygibson**
Fixed the javadoc issues in examples:

### CODE REVIEW

**Review by chkal — APPROVED** [review]
LGTM. Thanks!

**Review by andymc12 — APPROVED** [review]
LGTM - thanks!

File: examples/src/main/java/jaxrs/examples/bootstrap/ClientAuthenticationJavaSeBootstrapExample.java [hunk 1]
```diff
@@ -34,19 +34,15 @@
  * (e. g. 443, 8443, or someting completely different). The actual configuration needs to be queried after
  * bootstrapping, otherwise callers would be unaware of the actual chosen port. If the client's certificate is invalid
  * or cannot be validated, the server will reject the connection.
- * </p>
```
Comment (mkarg) [inline examples/src/main/java/jaxrs/examples/bootstrap/ClientAuthenticationJavaSeBootstrapExample.java:37]: I'm no HTML guru, but why is it wrong to use a closing tag here?

**Review by mkarg — APPROVED** [review]
Thanks a lot!

File: examples/src/main/java/jaxrs/examples/bootstrap/ClientAuthenticationJavaSeBootstrapExample.java (hunk 1 above)
Comment (andygibson) [reply examples/src/main/java/jaxrs/examples/bootstrap/ClientAuthenticationJavaSeBootstrapExample.java:37]: Javadoc isn't pure HTML, it

**Pull request merged**